controller/upload_file.rst

```diff
@@ -142,12 +142,12 @@ Finally, you need to update the code of the controller that handles the form::
             $form->handleRequest($request);
 
             if ($form->isSubmitted() && $form->isValid()) {
-                /** @var UploadedFile $brochureFile */
+                /** @var UploadedFile|null $brochureFile */
                 $brochureFile = $form['brochure']->getData();
 
                 // this condition is needed because the 'brochure' field is not required
                 // so the PDF file must be processed only when a file is uploaded
-                if ($brochureFile) {
+                if ($brochureFile !== null) {
```

Comment:
```suggestion
                if (null !== $brochureFile) {
```

Reply:
Aren't Yoda conditions:
- a bad practice due to English way of naturally reading text?
- only useful when not using strong typing?
- not consistent cause only needed when using "equal to" comparisons and not for any other operators, where it actually changes the conditions and therefore creates two ways of writing the same kind of logical treatment?

JK, tho. We all know I'll update the PR, it's in the standards: https://symfony.com/doc/current/contributing/code/standards.html.

Only question is: how long will we keep fake good practices as a standard? All of a mystery to me. ;)